[qe] qenvs rebranding to mapt

adrianriobo opened the pull request:
This version fix issues with resources not being deleted properly due to spot machines stopped prematurely

praveenkumar commented on build-qe-images.sh:
```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+# Copy me to the crc code folder
```
May be put more details like how to run this script, what parameters it uses ..etc.?

adrianriobo replied:
ahah sorry that was a mistake, I have it locally to create the images, but if it is valuable I can add it on a different PR

adrianriobo committed:
[qe] qenvs rebranding to mapt

qenvs has been rebrading to mapt, this commit updates the refereces. Also this version includes a fix issues with resources not being deleted properly due to spot machines stopped prematurely

Signed-off-by: Adrian Riobo Lorenzo <ariobolo@redhat.com>
